src/components/widget.js

```diff
@@ -101,36 +93,58 @@ export const query = graphql`
       }
     }
   }
-
+  fragment EventsParagraphFragment on paragraph__events_widget {
```

Comment:
Can you explain how to do this? I'm thinking it's not possible because the Events component has to use a static query to fetch the WordPress data (in other words, I don't know where I would put the EventsParagraphFragment)

Reply:
I'm pretty sure Static Query can handle fragments. I think you would just stick it directly below or above your query in the Events component (within the graphql backticks). 

For now, we can leave this as is and I can always move it later during another pull request.